src/components/Labelled/Labelled.tsx

```diff
@@ -48,6 +57,17 @@ export default function Labelled({
     </div>
   ) : null;
 
+  const characterCountMarkup = showCharacterCount &&
+    maxLength && (
+      <span
+        id={`${id}-labeled-character-counter`}
+        className={styles.CharacterCount}
+        aria-live="polite"
+      >
+        {`${characterCount} of ${maxLength} characters used`}
```

Comment:
This string will need to be externalized and internationalized. Some languages will change the position of words here, and we avoid hard-coding strings in our components. This is done using the `AppProvider` component and the `intl` context it provides. Strings live in the `en.json` file at the root of this project.

Grepping around the project should show you some examples of how that is done.